data-reconciliation-app/docker/ccf_app_js.enclave

```diff
@@ -0,0 +1,17 @@
+# Build
+FROM mcr.microsoft.com/ccf/app/dev:2.0.7-sgx as builder
+
+# Run
+FROM mcr.microsoft.com/ccf/app/run-js:2.0.7-sgx
+
+# copy configuration into image
+COPY ./governance/constitution/*.js /app/
+COPY ./workspace/docker_certificates/cchost_config_enclave_js.json /app/
```

Comment:
Given that the config files are now inside the same folder, there's no need to copy them. this was just the workaround so docker could access the files.

For this to work, the docker files should also be moved to the application folder, and not reside in the `docker` folder

to summarize:
today we have `root/docker/docker_file`
the new structure should be `root/application/docker_file`

Reply:
I believe structure "root/application/docker/docker_file " will work. 
Am I missing something here as I don't think docker_file needs to be moved inside application folder for this to work?

Reply:
Conceptually the docker files are on a specific folder and are making reference to the current scope `./workspace`, `./config` etc . However, as they are called from a different scope,  this reference to the current folder is not actually the one where the docker file resides.
So when someone reads the content of the docker file, it does not find the paths being referenced in the commands, resulting in a misleading interpretation. (there is no `workspace` folder inside the `docker` folder).


I personally don't mind leaving things like this, but this discussion has been raised previously and we need to make a decision towards it.

Either moving the docker files to the application root, or maybe just some comment explaining the correct context (the parent path) is enough.
Let's hear from others in the team of their opinions.

Reply:
@julioalex-rezende Could you try to resolve this by yourself? Merge is blocked because of "Changes requested" by you. I agree to put docker files for each app since it can handle application specific requirments in future and each app would look like independent project (therefore hopefully it would be easier to understand).

Reply:
As we have multiple docker files I prefer keeping them inside docker folder i.e. folder structure "root/application/docker/docker_files".

@ross-p-smith  let me know your thoughts on this.